package.json

```diff
@@ -122,10 +124,9 @@
     "xregexp": "^2.0.0"
   },
   "devDependencies": {
-    "babel-eslint": "^10.1.0",
+    "@babel/eslint-parser": "^7.17.0",
     "eslint": "^7.25.0",
     "eslint-config-apostrophe": "^3.4.0",
-    "eslint-loader": "^4.0.2",
```

Comment:
Deprecated and unused.
Fixes the `npm WARN deprecated eslint-loader@4.0.2: This loader has been deprecated. Please use eslint-webpack-plugin` warning

see https://github.com/webpack-contrib/eslint-loader